### PR TITLE
Jcf Map / it-academy.by / JC1-110-22 / Alena Trafimovich / Done tasks 1-3

### PR DESCRIPTION
**Pull request by AlenaTrafimovich**
## Pull request state

Actual status for PR:
- [x] implemented
- [x] reviewed
- [x] fixed
- [ ] approved

### CODE REVIEW

File: src/main/java/com/rakovets/course/java/core/practice/jcf_map/TextMonitoring.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
+package com.rakovets.course.java.core.practice.jcf_map;
+
+import java.util.*;
```
Comment (rakovets) [inline src/main/java/com/rakovets/course/java/core/practice/jcf_map/TextMonitoring.java:3]: Don't use * import

File: src/test/java/com/rakovets/course/java/core/practice/jcf_map/TextMonitoringTest.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+    @Test
+    public void testGetFrequencyWordsNegative() {
+        Map<String, Integer> actual = textMonitoring.getFrequencyWords(text, true);
+        List<Integer> valuesExpected = Arrays.asList(2, 1, 1, 3);
+        List<String> keysExpected = Arrays.asList("java", "is", "language", "Java");
+        Assertions.assertNotEquals(keysExpected, new ArrayList<>(actual.keySet()));
+        Assertions.assertNotEquals(valuesExpected, new ArrayList<>(actual.values()));
+    }
+}
```
Comment (rakovets) [inline src/test/java/com/rakovets/course/java/core/practice/jcf_map/TextMonitoringTest.java:65]: separate WHEN and THEN

**Review by rakovets — CHANGES_REQUESTED** [review]
Need to fix

File: src/main/java/com/rakovets/course/java/core/practice/jcf_map/TextMonitoring.java (hunk 1 above)
Comment (AlenaTrafimovich) [reply src/main/java/com/rakovets/course/java/core/practice/jcf_map/TextMonitoring.java]: fixed

File: src/test/java/com/rakovets/course/java/core/practice/jcf_map/TextMonitoringTest.java (hunk 2 above)
Comment (AlenaTrafimovich) [reply src/test/java/com/rakovets/course/java/core/practice/jcf_map/TextMonitoringTest.java]: fixed

**Pull request closed without merging**